src/api/images.tsx

```diff
@@ -11,13 +11,20 @@ import { EventQueue } from "context/eventQueue";
 import type { LxdInstance } from "types/instance";
 import type { UploadState } from "types/storage";
 import axios, { AxiosResponse } from "axios";
+import { withEntitlementsQuery } from "util/entitlements/api";
 
-export const fetchImageList = (project?: string): Promise<LxdImage[]> => {
+export const imageEntitlements = ["can_view"];
+
+export const fetchImageList = (
+  isFineGrained: boolean | null,
+  project?: string,
+): Promise<LxdImage[]> => {
+  const entitlements = `&${withEntitlementsQuery(isFineGrained, imageEntitlements)}`;
```

Comment:
`can_view` seems redundant. You won't see an image without that entitlement. Shall we remove entitlements for images?